torchtitan/logging.py

```diff
@@ -27,6 +27,9 @@ def init_logger(log_level):
     # suppress verbose torch.profiler logging
     os.environ["KINETO_LOG_LEVEL"] = "5"
 
+    # enable dataloading logging for logging the type of dataloading used
+    enable_dataloader_logging()
```

Comment:
Preferable to propagate log_level to this so we can easily get huggingface dataloader debug logs if needed.

Reply:
@tigranfah let's just get this to correspond to our log level, rest lgtm